### ISSUE TITLE
[Feature request]  Possibility to set "aspectRatio" in "CropRotateEditorState".

### ISSUE THREAD

**Issue by francescodinova** [hm21/pro_image_editor#169]
### Platforms

Android, iOS, Web, Windows, macOS

### Description

I would like to read `aspectRatio` attribute in `CropAreaHistory` mixin, and I would like a method to set it from code in `CropRotateEditorState`, so that it is added to the history and the state is updated accordingly (just as the `openAspectRatioOptions()` method does).

### Why

In this way, it is possible to create a completely custom aspect ratio selector for the crop editor.

**Comment by hm21** [comment]
Okay, I just released version `4.3.1` which now includes this feature. You can read and update the aspect ratio as below.

```dart
/// Read AspectRatio
double ratio = editorKey.currentState!.cropRotateEditor.currentState!.activeAspectRatio;

/// Update AspectRatio
editorKey.currentState!.cropRotateEditor.currentState!.updateAspectRatio(2 / 3);
```

**Comment by francescodinova** [comment]
Thanks it's perfect!
